examples/train_unconditional.py

```diff
@@ -27,8 +27,14 @@
 
 
 def main(args):
+    ddp_unused_params = DistributedDataParallelKwargs(find_unused_parameters=True)
```

Comment:
There are a couple of unused params in the new attention layer needed for checkpoint conversion, so we tell DDP to clean them up manually for now.